docs/developer/architecture.rst

```diff
@@ -317,7 +317,7 @@ by documentation for now, the rest is coming soon...).
    │   ├───experimental
    │   ├── :doc:`experimental </flow/modin/experimental/experimental>`
    │   │   ├───backends
-   │   │   │   └───omnisci
+   │   │   │   └─── :doc:`omnisci </flow/modin/experimental/backends/omnisci/index>`
```

Comment:
Please, add a leaf with query compiler rst as it is done for other backends.

Reply:
fixed